src/main/java/com/infernalstudios/greedandbleed/client/renderer/PigmyRenderer.java

```diff
@@ -17,7 +17,7 @@
 
 public class PigmyRenderer extends BipedRenderer<MobEntity, PigmyModel<MobEntity>> {
     private static final Map<EntityType<?>, ResourceLocation> resourceLocations = ImmutableMap.of(
-            EntityTypeRegistry.PIGMY.get(), new ResourceLocation(GreedAndBleed.MODID, "textures/entity/piglin/pigmy.png"));
+            EntityTypeRegistry.PYGMY.get(), new ResourceLocation(GreedAndBleed.MODID, "textures/entity/piglin/pigmy.png"));
```

Comment:
```suggestion
            EntityTypeRegistry.PYGMY.get(), new ResourceLocation(GreedAndBleed.MODID, "textures/entity/piglin/pygmy.png"));
```
Note: This change requires a filename change.